packages/clay-isomorphic/fixtures/expected/ClayTooltip.render

```diff
@@ -1 +1 @@
-<div class="tooltip fade clay-tooltip-bottom" data-onmouseenter="_handleMouseEnterTooltip" data-onmouseleave="_handleMouseLeave" role="tooltip"><div class="arrow"></div><div class="tooltip-inner">null</div></div>
\ No newline at end of file
+<div class="tooltip fade clay-tooltip-bottom" data-onmouseenter="_handleMouseEnterTooltip" data-onmouseleave="_handleMouseLeave"hiddenrole="tooltip"><div class="arrow"></div><div class="tooltip-inner">null</div></div>
```

Comment:
This doesn't look right... `"_handleMouseLeave"hiddenrole="tooltip"` 🤔

Reply:
@jbalsas thanks yes I forget the `{sp}`